src/routes/fest-route/fest-discussion.jsx

```diff
@@ -0,0 +1,102 @@
+import { useMutation } from "@apollo/client";
+import { useContext, useState } from "react";
+import { DiscussionCard } from "../../components";
+import { Button } from "../../components/button/index";
+import { CREATE_DISCUSSION, GET_DISCUSSIONS } from "../../graphql/index.js";
+import { CurrentUserContext } from "../../store/current-user-context.js";
+
+export const FestDiscussion = ({ discussionQuery, festQuery, festId }) => {
+  const [discussionContent, setDiscussionContent] = useState("");
+  const { currentUser } = useContext(CurrentUserContext);
+  const userId = currentUser?.id;
+  const isValid = discussionContent.length !== 0 ? true : false;
+
+  // Create an array of all attendees to later check if the current user is in list
+  const attendeesList = festQuery?.data?.fest?.attendees?.map(
+    (person) => person?.id
+  );
+
+  // Mutation to add discussion to server, and refetch the queries
+  const [
+    createDiscussion,
+    { discussionData, discussionIsLoading, discussionHasError },
+  ] = useMutation(CREATE_DISCUSSION, {
+    refetchQueries: [GET_DISCUSSIONS],
+  });
+
+  // Function to handle clicking of 'send' button; sends API response
+  const handleDiscussionSubmit = (evt) => {
```

Comment:
Message submits, is sent to the backend and displays correctly on the page immediately.

Reply:
No changes needed here right?

Reply:
Nope it looks good